workos/audit_logs.py

```diff
@@ -0,0 +1,126 @@
+import workos
+from workos.resources.audit_logs_export import WorkOSAuditLogsExport
+from workos.utils.pagiantion_order import Order
+from workos.exceptions import ConfigurationException
+from workos.resources.event import WorkOSEvent
+from workos.utils.request import RequestHelper, REQUEST_METHOD_GET, REQUEST_METHOD_POST
+from workos.utils.validation import AUDIT_TRAIL_MODULE, validate_settings
+
+EVENTS_PATH = "audit_logs/events"
+EXPORTS_PATH = "audit_logs/exports"
+
+
+class AuditLogs(object):
+    """Offers methods through the WorkOS Audit Logs service."""
+
+    @validate_settings(AUDIT_TRAIL_MODULE)
+    def __init__(self):
+        pass
+
+    @property
+    def request_helper(self):
+        if not getattr(self, "_request_helper", None):
+            self._request_helper = RequestHelper()
+        return self._request_helper
+
+    def create_event(self, organization, event, idempotency_key=None):
+        """Create an Audit Logs event.
+
+        Args:
+            organization (str) - Organization's unique identifier
+            event (dict) - An event object
+                event[action] (string) - The event action
+                event[version] (int) - The version of event
+                event[occurred_at] (datetime) - ISO-8601 datetime of when an event occurred
+                event[actor] (dict) - Describes the entity that generated the event
+                    event[actor][id] (str)
+                    event[actor][name] (str)
+                    event[actor][type] (str)
+                    event[actor][metadata] (dict)
+                event[targets] (list[dict]) - List of event targets
+                event[context] (dict) - Attributes of event context
+                    event[context][location] (str)
+                    event[context][user_agent] (str)
+                event[metadata] (dict) - Extra metadata
+            idempotency_key (str) - Optional idempotency key
+
+        Returns:
+            boolean: Returns True
+        """
+        payload = {"organization_id": organization, "event": event}
+
+        headers = {}
+        if idempotency_key:
+            headers["idempotency-key"] = idempotency_key
+
+        response = self.request_helper.request(
+            EVENTS_PATH,
+            method=REQUEST_METHOD_POST,
+            params=payload,
+            headers=headers,
+            token=workos.api_key,
+        )
+
+        return response["success"]
```

Comment:
Not sure if it's possible in Python, but we've been returning `Void` in the other SDKs as the `success` property is kind of a meaningless value that we return from various endpoints. There is some more context in this Slack thread that lead to this: https://work-os.slack.com/archives/C037Z7GMMAB/p1657562404232559

Reply:
So `void` in Python means `return None`. Nothing wrong with that, but I was trying to be consistent with other places in the Python SDK. 

https://github.com/workos/workos-python/blob/78f705670cfb76e8c9888e45a1148fea34ac0763/workos/passwordless.py#L70
https://github.com/workos/workos-python/blob/d5171cad529142b2470b5c69db47516776ac9b41/workos/audit_trail.py#L73

What do you think?

By the way, we don't have Python typing in our SDK (I want to propose that).

Reply:
Yeah it's going against historical convention in the other SDKs as well, I think we'll still want to make the change though to keep the various `createEvent` implementations the same.